filament/src/components/RenderableManager.cpp

```diff
@@ -205,6 +211,27 @@ RenderableManager::Builder& RenderableManager::Builder::enableSkinningBuffers(bo
     return *this;
 }
 
+RenderableManager::Builder& RenderableManager::Builder::boneIndicesAndWeights(size_t primitiveIndex,
+               math::float2 const* indicesAndWeights, size_t count, size_t bonesPerVertex) noexcept {
+    size_t vertexCount = count / bonesPerVertex;
+    utils::FixedCapacityVector<utils::FixedCapacityVector<filament::math::float2>> bonePairs(vertexCount);
+    for (size_t iVertex = 0; iVertex < vertexCount; iVertex++) {
+        utils::FixedCapacityVector<float2> vertexData(bonesPerVertex);
+        std::copy_n(indicesAndWeights + iVertex * bonesPerVertex,
+                    bonesPerVertex, vertexData.data());
+        bonePairs[iVertex] = std::move(vertexData);
+    }
+    return boneIndicesAndWeights(primitiveIndex, bonePairs);
+}
+
+RenderableManager::Builder& RenderableManager::Builder::boneIndicesAndWeights(size_t primitiveIndex,
+        utils::FixedCapacityVector<
+            utils::FixedCapacityVector<math::float2>> const &indicesAndWeightsVector) noexcept {
+    //mImpl->mBonePairs[primitiveIndex] = std::move(indicesAndWeightsVector);
+    mImpl->mBonePairs[primitiveIndex] = indicesAndWeightsVector;
+    return *this;
+}
```

Comment:
```suggestion
RenderableManager::Builder& RenderableManager::Builder::boneIndicesAndWeights(size_t primitiveIndex,
        utils::FixedCapacityVector<
            utils::FixedCapacityVector<math::float2>> indicesAndWeightsVector) noexcept {
    using std::swap;
    swap(mImpl->mBonePairs[primitiveIndex],  indicesAndWeightsVector);
    return *this;
}
```

If you use the `const&` parameter we are basically guaranteed to have a full copy of the vector each time. However, if you pass the parameter by value, the situation is better because *is* the caller uses `std::move` there will be no copy at all. If the caller doesn't, then we still have a single copy, so it's not worse.

You need to change the prototype in the header too.